examples/kupo/docker-compose.yaml

```diff
@@ -1,15 +1,21 @@
 services:
   cardano-node:
-    build:
-      context: .
-      dockerfile_inline: |
-        FROM ghcr.io/intersectmbo/cardano-node:9.0.0
-        COPY examples/cardano-node-completely-sync.sh /bin/healthcheck.sh
-        RUN chmod +x /bin/healthcheck.sh
+    image: ghcr.io/intersectmbo/cardano-node:9.0.0
+    healthcheck:
+      test: ["CMD-SHELL", "curl -f 127.0.0.1:12788 || exit 1"]
```

Comment:
@aciceri we are doing it in a smarter way (I've looked that up in cardano-db-sync repo)